pyproject.toml

```diff
@@ -72,9 +72,6 @@ where = ["."]
 exclude = ["tests", "tests.*", "examples", "examples.*", "tools", "tools.*"]
 namespaces = false
 
-[tool.black]
-line-length = 120
```

Comment:
what is the default now?

Reply:
https://github.com/openvinotoolkit/nncf/blob/3e087cd8b40215cdb3cf8f98eb2907a04215acb5/pyproject.toml#L102-L103
Still 120